compile/reflectivityCalculation/makeCompileArgs.m

```diff
@@ -34,7 +34,10 @@
 ARGS_1_1.contrastScalefactors = coder.typeof(0,[1 maxArraySize],[0 1]);
 ARGS_1_1.contrastBulkIns = coder.typeof(0,[1 maxArraySize],[0 1]);
 ARGS_1_1.contrastBulkOuts = coder.typeof(0,[1 maxArraySize],[0 1]);
-ARGS_1_1.contrastResolutionParams = coder.typeof(0,[1 maxArraySize],[0 1]);
+ARG = coder.typeof(0,[1 5],[1 1]);
```

Comment:
Why is this `ARG = coder.typeof(0,[1 5],[1 1]);` and background is `ARG = coder.typeof(0,[1 5],[0 1]);`? I thought the contents were same?

Reply:
No, for a data background, you need to point to a data file that contains the background (in the `source` field). Whereas for a data resolution, the resolution is included with the general data, so the `source` field is empty in this case.